DatabaseDesign/Database_design_SQL

```diff
@@ -0,0 +1,80 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Salary`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Salary` (
+  `employee_id` INT NOT NULL,
+  `salary_amount` DOUBLE NOT NULL,
+  `start_date` DATE NOT NULL,
+  `end_date` DATE NOT NULL,
+  `incentive_amount` DOUBLE NULL,
+  `bonus_amount` DOUBLE NULL,
+  `over_time_amount` VARCHAR(45) NULL,
+  UNIQUE INDEX `Employee ID_UNIQUE` (`employee_id` ASC),
+  UNIQUE INDEX `Date_UNIQUE` (`start_date` ASC),
+  UNIQUE INDEX `End_Date_UNIQUE` (`end_date` ASC))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Designation`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Designation` (
+  `employee_id` INT NOT NULL,
+  `post_name` VARCHAR(45) NOT NULL,
+  `post_details` VARCHAR(45) NULL,
+  `status` ENUM('Training','Probation','Actively-working','Terminated') NULL)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Employee`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Employee` (
+  `employee_id` INT NOT NULL,
+  `department_id` INT NOT NULL,
+  `first_name` VARCHAR(30) NOT NULL,
+  `middle_name` VARCHAR(30) NULL,
+  `last_name` VARCHAR(30) NOT NULL,
+  `employee_description` VARCHAR(45) NULL,
```

Comment:
I don't believe `VARCHAR(45)` will be enough for `employee_description`. Please be sure.